### ISSUE TITLE
Send Jenkins notifications if e2e job fails

### ISSUE THREAD

**Issue by ljelinkova** [fabric8io/fabric8-test#1182]
Modify Jenkins jobs to send email notofications like for BDD

https://github.com/openshiftio/openshiftio-cico-jobs/pull/904/files

**Comment by ppitonak** [comment]
I agree but would prefer to send notification to a mailing list instead of listing recipients
